docs/chat_managers/botpress_v12.md

```diff
@@ -2,11 +2,42 @@
 
 You can use the AAQ endpoints through various chat managers. Below is an example of how to get Botpress v12 (OSS) running and connected to AAQ endpoints using a provided demo bot.
 
-!!! note "[Botpress v12](https://github.com/botpress/v12/) is open-source and available to self-host but [Botpress Cloud](https://botpress.com/) is a completely different closed-source product."
+Note: [Botpress v12](https://github.com/botpress/v12/) is open-source and available to self-host but [Botpress Cloud](https://botpress.com/) is a completely different closed-source product.
 
-## Installation
+## Deployment
 
-### Via Docker
+### Option 1 - Via Docker Compose (behind NGINX with HTTPS)
+
+**Step 1:** Navigate to `chat_managers/botpress_v12/deployment/`
+
+**Step 2:** Copy `template.env` to `.env` and edit it to set the variables
+
+    PGPORT=
+    POSTGRES_DB=
+    POSTGRES_PASSWORD=
+    POSTGRES_USER=
+    ...
+
+**Step 3:** Copy `template.env.nginx` to `.env.nginx` and edit it to set the variables
+
+    DOMAIN=
+
+**Step 4:** Run `init-letsencrypt.sh` to get an SSL certificate from LetsEncrypt
```

Comment:
At some stage, I assume we will also provide default "bots" that will be loaded that setup the basic flow and link to whatsapp etc. I imagine we need the whatsapp connector sorted first.